### ISSUE TITLE
Paid Update

### ISSUE THREAD

**Issue by cyberfab007** [dwijitsolutions/laraadmin-crm#9]
I really like what these guys did and I use it for several projects , I am going to get a quote on how much it would cost to fork the project and update it laravel 5.5. Would anyone else be interested in this  ? I can't see it costing more then $500 dollars . 

Lend me y'all thoughts, I could do it my self I am in the middle of 6 projects right now. Two of the newer projects I wanted to use this backend but its not up to date so I am exploring paying some of my developers in India to just update it for us.

**Comment by cyberfab007** [comment]
I have officially forked the project , any help from the community would be appreciated

**Comment by saurabhd** [comment]
@cyberfab007 Don't think it would be an issue as soon as dependencies has been managed through composer..

**Comment by cyberfab007** [comment]
Yegh I got a quote from a guy already about 150 bucks to do it

**Comment by cyberfab007** [comment]
Actually I am gonna give him 250 cause I am a nice guy lol he should be commenting here soon,

**Comment by judevcoder** [comment]
@cyberfab007 hired me.

**Comment by cyberfab007** [comment]
@softdevstorm  How much you want to do the work ?

**Comment by judevcoder** [comment]
What do you mean?

**Comment by cyberfab007** [comment]
Make this project as is compatible with laravel 5.5 LTS and we go from there :) I am getting quotes in the $200-500 range and a week to complete  @softdevstorm

**Comment by judevcoder** [comment]
Please talk on skype

**Comment by cyberfab007** [comment]
@saurabhd I am not quite sure , I have not dived into the actual package , I was able to use this as is and build nice things off of it, I am sure there must be Syntexual differences in Laravel 5.5 like for instance how it handles routes . Someone will do this for me anyways I really dont have the time to many things going on. But its gotta get done

**Comment by judevcoder** [comment]
I couldn't hear from you for 4 days and access to your vps server.
Also I can't continue work on upwork, so I'd like to work outside of upwork.
If you agree with my proposal, I'll complete in this week.

Thanks

**Comment by saurabhd** [comment]
@softdevstorm @cyberfab007 Please respect the open source community and its guideline, any monetary terms should be discussed in person, not on public forum!

**Comment by judevcoder** [comment]
Hi, Jeremy

How are you?

**Comment by cyberfab007** [comment]
@saurabhd  Yes I am sorry , I did not realize this was the guy from upwork , my apologies 1000%

**Comment by cyberfab007** [comment]
@softdevstorm  sorry brother you were blocked on up work and just did not have the skill set needed to complete this , please move on :)

**Comment by judevcoder** [comment]
Hi

**Comment by judevcoder** [comment]
Did you recover your VPS server?
How can I update the version on the server without server

**Comment by judevcoder** [comment]
I launched the project on the server, but as I told you before, I can't access to the server.
Please visit to 45.77.137.226
